impl/src/main/java/org/apache/myfaces/cdi/model/FacesDataModelManager.java

```diff
@@ -64,6 +64,11 @@ public void init()
         }
     }
 
+    public void reinitialize()
```

Comment:
maybe better call it reset and just do: facesDataModels = null;

Reply:
Done